tests/SideBySide/CancelTests.cs

```diff
@@ -228,6 +228,18 @@ public void DapperQueryMultiple()
 			}
 		}
 
+		[Fact]
+		public async Task CancelCompletedCommand()
```

Comment:
I'm not getting any kind of failure from this test when running it without the new code (so it's difficult to verify that the problem is fixed).